docs-v2/reference/api/rate-limits.mdx

```diff
@@ -0,0 +1,13 @@
+---
+title: 'Rate limits'
+---
+
+The Nango API is rate-limited to prevent abuse and ensure fair usage across all clients. The rate limit is enforced on a per-account basis, with a fixed window of time and a maximum number of requests allowed within that window.
+
+Those headers are sent back with every API response:
+- `X-RateLimit-Limit` indicates the maximum number of requests a client can make within the rate limit window.
+- `X-RateLimit-Remaining` shows how many requests are remaining before the limit is reached.
+- `X-RateLimit-Reset` provides a Unix timestamp representing the time at which the current rate limit window resets, and the remaining request count is replenished.
+
+If a client exceeds the rate limit, the API will respond with a 429 `Too Many Requests` status code. In this case, the `Retry-After` header is included, indicating the number of seconds the client should wait before making another request to avoid being rate-limited.
```

Comment:
No documentation about the actual limit?

Reply:
The point of the headers is to specify the actual limit.  It is easier not to mention the limit imho so docs doesn't have to be changed when we modify the limit or if we start implementing different limits for different endpoints/group of endpoints.